apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -0,0 +1,52 @@
+const SET_LOGIN_TYPE = 'manageStudents/SET_LOGIN_TYPE';
+const SET_STUDENTS = 'manageStudents/SET_STUDENTS';
+const SET_SECTION_ID = 'manageStudents/SET_SECTION_ID';
+
+export const setLoginType = loginType => ({ type: SET_LOGIN_TYPE, loginType });
+export const setSectionId = sectionId => ({ type: SET_SECTION_ID, sectionId});
+export const setStudents = studentData => ({ type: SET_STUDENTS, studentData });
+
+const initialState = {
+  loginType: '',
+  studentData: [],
+  sectionId: null,
+};
+
+export default function manageStudents(state=initialState, action) {
+  if (action.type === SET_LOGIN_TYPE) {
+    return {
+      ...state,
+      loginType: action.loginType,
+    };
+  }
+  if (action.type === SET_SECTION_ID) {
+    return {
+      ...state,
+      sectionId: action.sectionId,
+    };
+  }
+  if (action.type === SET_STUDENTS) {
+    return {
+      ...state,
+      studentData: action.studentData,
+    };
+  }
+
+  return state;
+}
```

Comment:
Would be good to also include some reducer tests (even tho this is clearer pretty simple so far)